src/mapInfoContributorManager.cpp

```diff
@@ -91,10 +91,10 @@ MapInfoProperties MapInfoContributorManager::shortInfo(int roomID, int selection
     TRoom* room = mpHost->mpMap->mpRoomDB->getRoom(roomID);
     if (room) {
         QString areaName = mpHost->mpMap->mpRoomDB->getAreaNamesMap().value(areaId);
-        infoText = QStringLiteral("%1 (%3)\n")
-                           .arg(!room->name.isEmpty() && room->name != QString::number(room->getId()) ? QStringLiteral("%1/%2").arg(room->name, QString::number(room->getId()))
-                                                                                                      : QString::number(room->getId()),
-                                areaName);
+        auto roomName = room->name.remove(QRegularExpression("[.,/]$")).trimmed();
```

Comment:
Hm oh it's good you split this up into multiple lines for improved readability!
But I was talking about punctuation at the end of the name, not remove all inside..!

Reply:
Consider a room # 23 with a name like "The 2. floor / broom room." 
Which is now shown in short info like "The 2. floor / broom room./23".
With this PR too much will be removed: "The 2 floor  broom room/23" 
However should really only be like: "The 2. floor / broom room/23"

Reply:
Hmm after reading Regular Expression more carefully, this seems to do what's expected.
Trying to run the code now instead of only reading it :D

Reply:
No probs :)

Reply:
OK so you are correct and this only deletes the punctuation at the end of the line. 👍 
Also I applaud the extra space characters around the slash seperating the room number! 👍 

However, for some reason this also removes that punctuation from "full" map info?!
Did not find in code why that would be, and did not expect this happening. Did you?

![image](https://user-images.githubusercontent.com/117238/107992675-79fad480-6fd9-11eb-9de3-94f6778cd693.png)

Reply:
This will also remove repeated punctuation at the end, which is very 👍 as well 😄 

![image](https://user-images.githubusercontent.com/117238/107992839-d78f2100-6fd9-11eb-89b0-aa7fc60e1f7d.png)

Reply:
I guess I am not working on copy of variable, my bad... 
Repeated punctuation actually was not my target, thought that if repeated must be important somehow.. :) Wille take another look on that.

Reply:
Well, this was an elipsis .. ending the name, so only removing one of the dots would be ... strange! 
I say keep it, but let's wait for the others naming their taste in this matter

Reply:
Fixed. Working on copy right now. So far I've modified regex to remove multiple occurrences of characters.